ios/testng-examples/src/test/java/com/browserstack/AppiumTest.java

```diff
@@ -0,0 +1,27 @@
+package com.browserstack;
+
+import java.net.URL;
+
+import org.openqa.selenium.MutableCapabilities;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import io.appium.java_client.ios.IOSDriver;
+import io.appium.java_client.ios.options.XCUITestOptions;
+
+
+public class AppiumTest {
+
+    public IOSDriver driver;
+
+    @BeforeMethod(alwaysRun=true)
+    public void setUp() throws Exception {
+        MutableCapabilities options = new XCUITestOptions();
```

Comment:
use XCUITestOptions instead of MutableCapabilities

Reply:
+ @mattonem

Reply:
@avinash-bharti @kamal-kaur04 
As discussed in slack with both of you, can we close that comment?

Reply:
@mattonem lgtm, can close.